lib/active_merchant/billing/gateways/worldpay.rb

```diff
@@ -217,7 +229,7 @@ def add_payment_method(xml, amount, payment_method, options)
             end
           end
         else
-          xml.tag! 'paymentDetails' do
+          xml.tag! 'paymentDetails', add_attribute_for_cft_if_credit(options) do
```

Comment:
Sorry, this is very much a nitpick... how would you feel about a method name of:

`credit_fund_transfer_attribute(options)`?